credsweeper/utils/util.py

```diff
@@ -73,8 +73,23 @@ def get_shannon_entropy(cls, data: str, iterator: Chars) -> float:
 
     @classmethod
     def read_patch_file(cls, path: str) -> List[str]:
```

Comment:
It would be nice to add docstring about supported formats and description on how this function chose between them